src/sanitizer.js

```diff
@@ -320,6 +339,12 @@ export function isValidAttr(tagName, attrName, attrValue) {
     return false;
   }
 
+  if (attrName == 'class' &&
+      attrValue &&
+      /(^|\W)i-amphtml-/i.test(attrValue)) {
+    return false;
+  }
```

Comment:
We don't allow the `class` attribute, since we dropped it from `WHITELISTED_ATTRS`?